### ISSUE TITLE
Shorter names for vis_filepath and vis_filename

### ISSUE THREAD

**Issue by jgarte** [martanne/vis#1013]
What do you think of having shorter names for [vis_filepath and vis_filename](https://github.com/martanne/vis/blob/1a958f221404b09cb8b0612fb34301e6b9783cf9/vis.c#L1809)?

I was thinking `fp` instead of `vis_filepath` and `fn` instead of `vis_filename`

See the [wiki](https://github.com/martanne/vis/wiki/Tips-and-Tricks#how-can-i-spellcheck-the-current-file) for ways that these environment variables are used.

wdyt

**Comment by ninewise** [comment]
I'd rather not add these. The current names are less confusing and I don't expect anyone to write these so often the length matters.

**Comment by jgarte** [comment]
I have to agree with @mcepl here. I would be using it all the time if it were shorter like in vim.

I think I will just patch vis when I get around to it. I've been too lazy to do it.

**Comment by mcepl** [comment]
No need for fork, we are trying to be active here. @ninewise looks over pull requests quite well.

**Comment by jgarte** [comment]
By fork I meant to to just patch vis to do what I need not to turn it into some other project ;)

**Comment by lobre** [comment]
Just dropping a quick comment to show my interest. I think having `%` as the "current file name" register could make sense. It aligns well with vim also.

No promise, but if the idea makes sense, it could be a good beginner patch that I can try to implement.
